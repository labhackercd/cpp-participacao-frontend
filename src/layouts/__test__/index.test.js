import React from 'react';
import { mount } from 'enzyme';
import MockTheme from '../../mocks/theme/mockTheme';
import Layout from '../index';

test('Snapshot Layout page should not have changes', () => {
  const component = mount(<MockTheme><Layout /></MockTheme>);
  expect(component.exists()).toEqual(true);
  expect(component).toMatchSnapshot();
});
