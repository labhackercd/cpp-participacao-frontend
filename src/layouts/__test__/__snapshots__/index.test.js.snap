// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Layout page should not have changes 1`] = `ReactWrapper {}`;
